File: AI/M_UCT/M_GameTree.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SimpleWars
{
	/// <summary>
	/// M_UCTに使用されているゲームツリー
	/// </summary>
    class M_GameTree
    {
        public Map board;
        public int depth;
        public int simnum;
        public double lastscore;
        public double totalscore;
        public double housyuu;
        public Action act;
        public List<M_GameTree> next;

        public M_GameTree()
        {
            board = new Map();
            depth = 0;
            simnum = 0;
            lastscore = 0;
            totalscore = 0;
            housyuu = 0;
            act = new Action();
            next = new List<M_GameTree>();
        }

    }
}

File: Taylor/port/MGameTree.py
from Action import Action
from Map import Map

class MGameTree:
    def __init__(self):
        # self.board = Map() 

        self.depth = 0
        self.simnum = 0
        self.last_score = 0.0
        self.total_score = 0.0
        self.housyuu = 0.0  # Presumably some kind of reward or score
        self.act = Action()  # Assumed to be an object similar to the C# 'Action'
        self.next = []  # List of MGameTree objects (child nodes)